.config/openbox/rc.xml

```diff
@@ -334,9 +334,9 @@
       <action name="Execute">
         <startupnotify>
           <enabled>true</enabled>
```

Comment:
It might be safer to set `false` just in case the file manager doesn't support startup notifications?